nifi-extension-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/test/java/org/apache/nifi/processors/hadoop/GetHDFSTest.java

```diff
@@ -226,12 +225,6 @@ public void testDirectoryUsesValidEL() throws IOException {
         assertEquals("13545423550275052.zip", flowFile.getAttribute(CoreAttributes.FILENAME.key()));
         InputStream expected = getClass().getResourceAsStream("/testdata/13545423550275052.zip");
         flowFile.assertContentEquals(expected);
-        final List<ProvenanceEventRecord> provenanceEvents = runner.getProvenanceEvents();
```

Comment:
Thanks for the improvements @exceptionfactory, is there any reason why the provenance event assertion part was removed from the test, as I can see there is no other test containing this check?

Reply:
Thanks for noting that @mark-bathori, you're right that no other test includes the Provenance Event checking. I restored that check in the test method.